### ISSUE TITLE
First name and last name should be merged to one field

### ISSUE THREAD

**Issue by wormyy** [Covid19-SK/triage-app#38]
![Screenshot 2020-03-20 at 00 08 20](https://user-images.githubusercontent.com/8556724/77122852-f2aa8700-6a3e-11ea-89fd-c31dc5458078.png)

**Comment by marty-buly** [comment]
Dobre tomu rozumiem, ze tento ticket si vyzaduje zmenu na frontend strane v sekciach "registration" a "admin/patients", plus zmenu na backend pre "patients". Ak je to takto zoberiem si to na starost.
Zabudol som este na nieco?

**Comment by mkormos** [comment]
Ano, ide o to, ze fieldy "firstName" a "lastName" sa zmerguje do jedneho (napr. "name"). Cize zmena je potrebna na frontende aj backende, aj v UI.

**Comment by marty-buly** [comment]
Tak ked sa potvrdi, ze tuto apku treba, ja si to vezmem na starost

**Comment by marty-buly** [comment]
PR created https://github.com/Covid19-SK/triage-app/pull/40
